doc/flame/inputs/drag_events.md

```diff
@@ -175,15 +175,15 @@ class MyComponent extends PositionComponent with DragCallbacks {
 ```
 
 
-### HasDraggablesBridge
+### HasDraggableBridge
```

Comment:
Faulty change, `HasDraggables` and `HasDraggablesBridge` are the names of the mixins.

Reply:
oops my bad